alf/environments/suite_dmlab.py

```diff
@@ -206,6 +209,9 @@ def load(scene,
     Returns:
         A PyEnvironmentBase instance.
     """
+    global _unwrapped_env_checker_
```

Comment:
no need to have this?

Reply:
confirmed with @witwolf , dmlab can only create one env, too

Reply:
removed